src/rippled/transactions.js

```diff
@@ -8,11 +8,14 @@ const log = logger({ name: 'transactions' })
 const getTransaction = (transactionId, rippledSocket) => {
   log.info(`get tx: ${transactionId}`)
   return getRippledTransaction(rippledSocket, transactionId)
-    .then((response) => formatTransaction(response))
-    .then((data) => ({
-      summary: summarizeTransaction(data, true).details,
-      raw: data,
-    }))
```

Comment:
In the current codebase, `raw == formatTransaction(response)`. In the PR, you have directly invoked this function, instead of chaining two callbacks.

Is there any difference between `raw` in the current codebase and `processed` variable in the PR? Why is the change working? 😅